apps/heft/src/utilities/subprocess/SubprocessTerminator.ts

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import * as childProcess from 'child_process';
+import process from 'process';
+
+/**
+ * When a child process is created, registering it with the SubprocessTerminator will ensure
+ * that the child gets terminated when the current process terminates.
+ *
+ * @remarks
+ * This works by hooking the current process's events for SIGTERM/SIGINT/exit, and ensuring the
+ * child process gets terminated in those cases.
+ *
+ * SubprocessTerminator doesn't do any thing on Windows, since by default Windows automatically
```

Comment:
"any thing" instead of "anything" seems odd to me, but is probably also correct.

Reply:
I think you're right - "anything" sounds more right.

```suggestion
 * SubprocessTerminator doesn't do anything on Windows, since by default Windows automatically
```

Reply:
Fixed